### PR TITLE
Cdb 267 disable controls

### PR DESCRIPTION
**Pull request by Xatpy**
Issue related [267](https://github.com/CartoDB/cartodb.js/issues/267)

*zoomControl* was no working. I fixed it, and write tests (I tried to they were unit tests)

Also, there was an error about naming in the documentation. It's search, and no searchControl (I guess it was an error because zoomControls was near).

Please, @javisantana , could you review it?

### CODE REVIEW

File: doc/API.md [hunk 1]
```diff
@@ -192,7 +192,7 @@ cartodb.createVis('map', url)
   - **shareable**: add facebook and twitter share buttons.
   - **title**: adds a header with the title of the visualization.
   - **description**: adds description to the header (as you set in the UI).
-  - **searchControl**: adds a search control (default: false).
+  - **search**: adds a search control (default: true).
```
Comment (javisantana) [inline doc/API.md]: should this change go in this PR ?

**Comment by Xatpy** [comment]
@javisantana I updated documentation here, because there's no doc about cartodb.js in CartoDB/docs repo.

What do you think about last commit? I'm not completely sure about [this](https://github.com/CartoDB/cartodb.js/commit/511d369504443313c2129962779ef5dd464caee4#diff-f99ed960498d5ea8717b3e74d54bfb01R812) (specifically in `this.mobile`)

**Comment by Xatpy** [comment]
please @javisantana , what do you think about this pr?

**Comment by javisantana** [comment]
:es:

**Pull request merged**